xwayland/selection/outgoing.c

```diff
@@ -297,7 +297,6 @@ static bool xwm_selection_send_data(struct wlr_xwm_selection *selection,
 		return false;
 	}
 
-	xwm_selection_transfer_init(transfer);
```

Comment:
Temporarily removed because this is the outgoing path, which has no logic to release the X11 window yet (so this would just leak).

Reply:
Maybe replace with `transfer->wl_client_fd = -1` temporarily, and add a TODO comment about it?

Reply:
Sure, I've done that. I didn't originally because I thought the outgoing path didn't do comparisons with 0, but I was wrong and it does.